src/components/IOUConfirmationList.js

```diff
@@ -132,9 +132,15 @@ class IOUConfirmationList extends Component {
     /**
      * Gets splits for the transaction
      *
-     * @returns {Array}
+     * @returns {Array|null}
      */
     getSplits() {
+        // There can only be splits when there are multiple participants, so return early when there are not
+        // multiple participants
+        if (!this.props.hasMultipleParticipants) {
+            return null;
```

Comment:
Any reason why this wouldn't return an empty object instead?

Reply:
I originally wrote this with it returning an empty array... but then I changed it for some particular reason that I am having trouble recalling. I think I did it because it makes the `createTransaction()` method a little cleaner because you can do `if (splits) {` rather than `if (splits.length) {`